test_framework/terraform/harvester/sles/main.tf

```diff
@@ -0,0 +1,208 @@
+terraform {
+  required_providers {
+    rancher2 = {
+      source  = "rancher/rancher2"
+      version = "~> 4.4.0"
```

Comment:
Can we pivot to [v5.1.0](https://github.com/rancher/terraform-provider-rancher2/releases/tag/v5.1.0) latest version? v5.x has some fixes.

Reply:
Refined.